python/ray/tune/logger.py

```diff
@@ -316,6 +316,58 @@ def close(self):
         self._file.close()
 
 
+class TBXLogger(Logger):
+    """TensorBoardX Logger, because TF breaks everything.
+
+    Automatically flattens nested dicts to show on TensorBoard:
+
+        {"a": {"b": 1, "c": 2}} -> {"a/b": 1, "a/c": 2}
+    """
+
+    def _init(self):
+        from tensorboardX import SummaryWriter
```

Comment:
Should we make this a mandatory dep?

Reply:
No I think not. It has issues of its own...